.tekton/operator-next-bundle-pull-request.yaml

```diff
@@ -45,7 +45,9 @@ spec:
     value: "true"
   - name: prefetch-input
     value: |
-      {"type": "rpm", "path": ".konflux/rpms"}
+      [ {"type": "rpm", "path": ".konflux/rpms"},
+        {"type": "generic", "path": ".konflux/rpms"}
```

Comment:
Why re-using `.konflux/rpms` ? it should be something else, it has nothing to do with rpms.